AlmaLinux-8-RaspberryPi-gnome.aarch64.ks

```diff
@@ -26,9 +26,9 @@ lang en_US.UTF-8
 
 # Disk setup
 clearpart --initlabel --all
-part /boot --asprimary --fstype=vfat --size=500 --label=boot
```

Comment:
Please remove the size change, it cannot be accepted, to keep the size of download image small.